PMMain/Midas/MidasGUI/GestureFilter.cpp

```diff
@@ -96,8 +96,8 @@ void GestureFilter::registerMouseSequences(void)
     clickResp.responseType = ResponseType::MOUSE_CMD;
     clickResp.responseAction.mouse = mouseCmds::LEFT_CLICK;
     int ss = (int)gestSeqRecorder.registerSequence(midasMode::MOUSE_MODE, clickSeq, clickResp);
-    clickSeq.at(0).poseLen = SeqElement::PoseLength::TAP; // Jorden. TODO. Not quite this, or it can 'stick on click'
-    ss |= (int)gestSeqRecorder.registerSequence(midasMode::GESTURE_MODE, clickSeq, clickResp);
+    //clickSeq.at(0).poseLen = SeqElement::PoseLength::TAP; // Jorden. TODO. Not quite this, or it can 'stick on click'
+    //ss |= (int)gestSeqRecorder.registerSequence(midasMode::GESTURE_MODE, clickSeq, clickResp);
```

Comment:
Why are these lines commented out? Is this for something that you will be doing in the future, or should these just be removed?